spec/requests/questions_spec.rb

```diff
@@ -0,0 +1,135 @@
+require 'rails_helper'
```

Comment:
I am assuming this is auto generated via rails.. have you tried it out? What are the results?

Reply:
auto generated